webforces/api/views.py

```diff
@@ -116,15 +118,36 @@ class AlgsView(APIView):
 
     def get(self, request):
         core = Core()
-        status, algs = core.db.getAllAlgs()
+        if request.data['fromTitle'] == '1':
```

Comment:
What does `fromTitle` param mean? We already have separate view for getting algs using title. See `class AlgViewTitle` in current file.